packages/sdk/src/util/utils.ts

```diff
@@ -84,8 +85,14 @@ export function getTenantIdAndLoginHintFromSsoToken(ssoToken: string): any {
 /**
  * @internal
  */
-export function parseAccessTokenFromAuthCodeTokenResponse(tokenResponse: string): AccessToken {
+export function parseAccessTokenFromAuthCodeTokenResponse(
+  tokenResponse: string | AuthenticationResult
+): AccessToken {
   try {
+    if (typeof tokenResponse == "object") {
```

Comment:
can we: 
```
if (typeof tokenResponse == "string") {
 JSON.parse(tokenResponse)
}
```?
it may be strange that we parse the json string right after stringify the object.

Reply:
Fixed. Thx!